### DISCUSSION TITLE
Drag and drop windows with key?

### DISCUSSION THREAD

**Discussion by marc-hg** [dalyIsaac/Whim#866] [Q&A] [answered]
One thing I like doing in i3 and yabai is being able to move windows without clicking the topbar but doing something like super + left click and it moves the window (floating or split)

is this possible in whim?

**Comment by urob** [comment]
Yes, that's what the [`swap_window_in_direction.*`](https://dalyisaac.github.io/Whim/docs/customize/commands.html) commands do. By default they are bind to `Win` + `Left|Right|Up|Down`.

**Reply by marc-hg** [reply]
I meant dragging into a fixed position in floating mode

**Reply by urob** [reply, marked as answer]
As far as I know, there's currently no way to move windows *while* in floating-mode on the Whim-side. The workarounds that come to mind are (1) create a Whim binding that untoggles floating and then moves the window (this would end up unfloated). (2) Or use the native Windows short-cuts to snap floated windows to one of the edges.

**Comment by airtonix** [comment]
AltSnap will let you do this. along with resize.

but whim doesn't care about altsnap so your window changes done this way will remain unoticed by whim.
